bin/phpunit-docker.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+WP_VERSION=${1-latest}
+
+MYSQL_ROOT_PASSWORD='wordpress'
+db=$(docker run -p 3306:3306 -e MYSQL_ROOT_PASSWORD=$MYSQL_ROOT_PASSWORD -d mariadb)
```

Comment:
Minor suggestion: If we avoid using the host network, then there won't be a conflict if MySQL is already running there.
```suggestion
docker network create tests
db=$(docker run -p 3306:3306 --network tests --name db -e MYSQL_ROOT_PASSWORD=$MYSQL_ROOT_PASSWORD -d mariadb)
```